runtime/doc/news.txt

```diff
@@ -123,7 +123,8 @@ BREAKING CHANGES IN HEAD                                    *news-breaking-dev*
 The following breaking changes were made during the development cycle to
 unreleased features on Nvim HEAD.
 
-• ...
+• Removed `vim.treesitter.foldtext` as transparent foldtext is now the default
+  https://github.com/neovim/neovim/pull/20750
```

Comment:
Nit: it's not default, you need to set `foldtext=''` explicitly.

Reply:
In my head `foldtext=''` was the default, but you are right, I changed it.

Reply:
It is a good candidate to _become_ a default though